client/lib/plans/features-list.js

```diff
@@ -217,17 +217,15 @@ export const FEATURES_LIST = {
 		getSlug: () => constants.FEATURE_UNLIMITED_STORAGE_SIGNUP,
 		getTitle: () => i18n.translate( '200 GB storage' ),
 		getDescription: () =>
-			i18n.translate(
-				"With increased storage space you'll be able to upload more images, videos, audio, and documents to your website."
-			),
+			i18n.translate( 'Upload more images, videos, audio, and documents to your website.' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'With increased storage space you\'ll be able to upload more images, videos, audio, and documents to your website.' )` *ES Score: 6*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plan-page-tooltip-copy)